resources/views/components/overview.blade.php

```diff
@@ -0,0 +1,8 @@
+<div {{ $attributes->class('text-primary flex flex-wrap gap-x-8 text-sm max-md:flex-col') }}>
+    <div {{ $main->attributes->class('flex-1') }}>
+        {{ $main }}
+    </div>
```

Comment:
```suggestion
    <div class="flex-1">
        {{ $slot }}
    </div>
```

With this you don't have to use `<x-slot:main>...</x-slot:main>`, currently no attributes are added and if you need something else; just use it within the slot:

```
<x-rapidez-ct::overview>
    <div class="bg-red">
        Left
    </div>

    <x-slot:sidebar>
        Right
    </x-slot:sidebar>
</x-rapidez-ct::overview>
```

Maybe rename this to something else, maybe layout? With an optional sidebar?